src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,155 @@
+package duke;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * this class contains the task list
+ * it will handle all task commands
+ * @author Kaiyi
+ */
+public class TaskList {
+
+    ArrayList<Task> listOfInputs = new ArrayList<>();
+    private String directory;
+    private String filePath;
+
+    private Storage storage = new Storage(directory, filePath);
+    /**
+     * @param directory
+     * @param filePath
+     */
+    public TaskList (String directory, String filePath) {
+        this.directory = directory;
+        this.filePath = filePath;
+
+    }
```

Comment:
I think you should initialize storage field within constructor since it uses the parameters of constructor?